app/client/build.sh

```
@@ -1,8 +1,11 @@
 #!/bin/sh
 
 set -e
+if [ -z $GIT_SHA ]
```

In what scenarios will the GIT_SHA not be available to us when building the code? This sha is used to mark a new release in Sentry. Hence, should we let the build fail/crash if the GIT sha is not available? Else, there may be scenarios where even though the sha isn't avaiable, the build will succeed. That might cause more issues.

---

This can happen in the scenario where some engineer runs 

`docker build . -t fat_container`

instead of  `docker build --build-arg GIT_SHA=${GITHUB_SHA} . -t fat_container`.

In the Dockerfile we are not copying the `.git` folder. Rather, we take the sha in build args and bake it as an environment variable in the container. So, in case if the sha is not sent as part of build args the docker build it will fail.

To also let the engineers build client locally, using 

`yarn install --frozen-lockfile; yarn build`

 Where the GIT_SHA wont be there by default but the script will get the git_sha and wont fail.